meteor/client/styles/rundownView.scss

```diff
@@ -1114,6 +1116,18 @@ body.no-overflow {
 			}
 		}
 
+		.segment-timeline__liveline__apendage {
+			position: absolute;
+			margin-left: 5px;
+			margin-top: 0.1em;
+
+			&.segment-timeline__liveline__apendage--piece-countdown {
```

Comment:
Just to make sure I hold the record for the team's nittiest pick: _appendage_ is written with 2 p's :)

Reply:
Good find - fixed :)